app/models.py

```diff
@@ -1363,6 +1366,7 @@ class Token(Base):
     name = sa.Column(
         sa.String,
         nullable=False,
+        unique=True,
```

Comment:
I think this needs to get removed -- Token names can be identical for different users.